lib/PayPal/Api/Amount.php

```diff
@@ -52,7 +51,6 @@ public function getCurrency()
     public function setTotal($total)
     {
         NumericValidator::validate($total, "Total");
```

Comment:
Should `NumericValidator::validate` calls also be removed?

For instance, `is_numeric('42,123')` seems to return false.